lib/backend/leveldb.js

```diff
@@ -1017,7 +1015,6 @@ ConsistentHash.prototype.remapVnode = function remapVnode(newPnode, vnode, cb) {
                     return _cb();
                 }
             });
-            console.log('{"%s":"%s"}', newPnode, vnode);
```

Comment:
This might be relevant output for `./bin/fash.js remap-vnode`?  I'm not sure.

Reply:
agreed. will add back.